README.md

```diff
@@ -76,7 +76,7 @@ different letters.
 
 What is the minimum number of bits to be able to represent the English alphabet?
 
-** ADD YOUR ANSWER HERE **
+26 bits
```

Comment:
With 26 bits you could represent 2^26 combinations or 67,108,864 different values. How many bits do you need to represent 26 values?